apps/metadata-editor/src/app/dashboard/sidebar/sidebar.component.html

```diff
@@ -31,8 +31,14 @@
         <gn-ui-button
           type="default"
           class="w-10 h-10 flex justify-center items-center hover:cursor-pointer"
+          (click)="logOut()"
         >
-          <img src="assets/system-shut.svg" alt="Log out" class="w-4 h-4" />
+          <img
+            src="assets/system-shut.svg"
+            alt="Log out"
```

Comment:
I'm wondering if the alt text should also be translated?

Reply:
It was not translated so I didn't do it either, but sure, I don't see why not :slightly_smiling_face: